src/core/dbus.c

```diff
@@ -160,8 +160,9 @@ static int signal_activation_request(sd_bus_message *message, void *userdata, sd
                 return 0;
         }
 
-        if (!manager_dbus_is_running(m)) {
-                r = sd_bus_error_set(&error, BUS_ERROR_SHUTTING_DOWN, "Refusing activation, D-Bus is not running.");
+        if (manager_unit_inactive_or_pending(m, SPECIAL_DBUS_SERVICE) ||
```

Comment:
Hmm, this revert doesn't make a huge difference I think. We can't set up the handler for `ActivationRequest` if dbus isn't running after all 🤔

Reply:
I think this is exactly what we're trying to avoid in the first place? `Type=dbus` units pull in `dbus.{service,socket}`, and that would cause job type conflict...